apps/ledger-live-mobile/src/actions/settings.ts

```diff
@@ -167,6 +167,16 @@ export const setLastSeenDeviceInfo = (dmi: DeviceModelInfo) => ({
   type: "LAST_SEEN_DEVICE_INFO",
   payload: dmi,
 });
+
+export const setLastSeenDevice = ({
+  deviceInfo,
+}: {
+  deviceInfo: DeviceInfo;
+}) => ({
+  type: "LAST_SEEN_DEVICE",
+  payload: { deviceInfo },
+});
+
```

Comment:
Why do we need this when we have `setLastSeenDeviceInfo` just above? 🤔

Reply:
They're different types, `setLastSeenDeviceInfo` sets more info than only the deviceInfo it sets a DeviceModelInfo, and I'd have to modify the component that uses this only to extract this extra info and update it. I thought it would be better to just have a more granular action that updates only what we need to update.

That being said, this actions aren't really well named :/. I didn't want to refactor the name in this PR to not make it bigger than it needs to be, but if you think I should, I can do it.

Reply:
I don't mind